Descope/Types/Types.cs

```diff
@@ -674,4 +678,193 @@ public class PasswordSettings
         [JsonPropertyName("lockAttempts")]
         public int LockAttempts { get; set; }
     }
+
+    public class RoleMapping
+    {
+        public List<string> Groups { get; set; }
+        public string Role { get; set; }
+        public RoleMapping(List<string> groups, string role)
+        {
+            Groups = groups;
+            Role = role;
+        }
+    }
+
+    public class SsoSamlSettings
+    {
+        [JsonPropertyName("idpURL")]
+        public string IdpUrl { get; set; }

```

Comment:
Looking at the backend code, this isn't the correct JSON field name. If this was copied from some other SDK you might wanna check if there's a mistake in the original source code as well
```suggestion
        [JsonPropertyName("idpUrl")]
        public string IdpUrl { get; set; }
```